donut/static/js/groups/positions.js

```diff
@@ -127,3 +128,61 @@ function changePosition(posName) {
     $('#positionSelect').val(posName);
     filterChange();
 }
+
+/*
+ *
+ */
+function groupDelChange() {
+    var groupIndex =$('#groupDel').val();
+    $('#position').find('option').remove();
+    var url = '/1/groups/' + groupIndex + '/positions/';
+    $.ajax({
+        type: 'GET',
+        url: url,
+        success: function(data){
+            for (var i = 0; i < data.length; i++) {
+                var newOption = '<option value=' + data[i].pos_id + '>' + 
+                    data[i].pos_name + '</option>';
+                $('#position').append(newOption);
+            }
+        }
+    })
+}
+
+// Setting up triggers for administration tasks
+$(document).ready(function() {
+    $('#submitPosBtn').click(function(e) {
+        e.preventDefault();
+        $.ajax({
+            type: 'POST',
+            url: '/1/positions/',
+            data: $("#posCreate").serialize(),
+            success: function(data) {
+                p = JSON.parse(data);
+                if(p.success) {
+                    $('#posCreate').trigger("reset");
+                    alert("Position created!");
+                }
+            }
+        });
+    });
+    $('#delPosBtn').click(function(e) {
+        e.preventDefault();
+        $.ajax({
+            type: 'POST',
+            url: '/1/positions/delete/',
+            data: $("#posDelete").serialize(),
+            success: function(data) {
+                p = JSON.parse(data);
+                if(p.success) {
+                    $('#posDelete').trigger("reset");
+                    $('#position').find('option')
+                                .remove()
+                                .end()
+                                .append('<option>Select a Position</option>');
```

Comment:
Can put this "remove all position options and add `Select a position`" functionality in a function that gets called on page load, rather than also including it in the HTML